gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -239,21 +239,15 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         # Create SSL credentials for mutual TLS if needed.
         use_client_cert = bool(util.strtobool(os.getenv("GOOGLE_API_USE_CLIENT_CERTIFICATE", "false")))
 
-        ssl_credentials = None
+        client_cert_source = None
```

Comment:
Minor naming nit: maybe `client_cert_source_func` to make it clear that this is a callable?

Reply:
done